Makefile

```diff
@@ -18,14 +18,10 @@ clean:
 	rm ${OUT}
 # Please make sure that user has enough privilege to execute docker command
 docker: 
-	LIBRARY_PATH=/usr/local/lib go build -o ${BINARY} main.go
```

Comment:
why remove the build process when creating image?

Reply:
As https://github.com/istio-ecosystem/hsm-sds-server/pull/26#issue-1687834737 said, using different host os versions to build binary has different dependencies on libc, so make sure to compile and run using the same os.

Reply:
Okay, we may need to consider the `make release` for different OS versions in later, such as `ubuntu 20.04` or `22.04`, etc.